pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -685,6 +765,16 @@ func newUserAccountWithFinalizer(userName, userID string) *toolchainv1alpha1.Use
 	return userAcc
 }
 
+func newNSTmplSetSpec() toolchainv1alpha1.NSTemplateSetSpec {
+	return toolchainv1alpha1.NSTemplateSetSpec{
+		TierName: "basic",
+		Namespaces: []toolchainv1alpha1.Namespace{
+			{Type: "dev", Revision: "rev1", Template: ""},
+			{Type: "code", Revision: "rev1", Template: ""},
```

Comment:
may I know why you tend to use only two namespaces in the NsTemplateSet when the default configuration will contain three of them?

Reply:
One is less for edge cases, Two is enough for this scope of test.